pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -262,3 +263,46 @@ def run(self):
                            self._data.url,
                            Variant('a{sv}', remote_options),
                            cancellable)
+
+
+class ConfigureOSTreeBootloader(Task):
```

Comment:
I would rename this, I don't think there is such a thing as OSTree Boot Loader. Just a custom configuration of a boot loader for OSTree. Maybe ``ConfigureBootLoader`` should be fine and leave the OSTree in the ``name`` property?

Reply:
Good idea - done.